### DISCUSSION TITLE
URun not working on blooket

### DISCUSSION THREAD

**Discussion by Perplexedykwhat** [3kh0/ext-remover#1316] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Help. Whenever I try to open URun on blooket it gives me an error. (It says: This content is blocked by the owner of the site. Contact the owner to fix this.)Probably impossible to fix though. Keep in mind I am a skid.

**Comment by schoolexploitkid** [comment, marked as answer]
use the ublock run method

**Reply by Perplexedykwhat** [reply]
yo how do I set it up though, any links/ repository that I can see?

**Reply by schoolexploitkid** [reply]
yes, https://github.com/3kh0/ext-remover?tab=readme-ov-file#ublock-run--run-code-on-pages:~:text=Back%20to%20top-,uBlock%20Run%20Run%20Code%20On%20Pages,-Note

**Reply by Perplexedykwhat** [reply]
thanks, I'm going to try this when I get into school again

**Reply by schoolexploitkid** [reply]
also tbh dont hack blooket my account got banned for a month :( for getting like 500 coins every day

**Comment by nathblade16** [comment]
the unlock run method might work but keep in mind some websites have CSP that stops this from working all together, csp means content security policy and stops some xss exploits
